### ISSUE TITLE
openblas-devel missing lapack.h

### ISSUE THREAD

**Issue by sdrobert** [AnacondaRecipes/openblas-feedstock#6]
Issue: the `openblas-devel` package ought to contain all reference LAPACKE headers, including "lapack.h." As of the commit f55b188bd851171be55ade5687e6247844b8672f (I think), "lapack.h" is no longer included in the install. "lapacke.h" includes "lapack.h" in its definition, which will result in a missing header error if included.

```
$ ls ${CONDA_PREFIX}/include
cblas.h  f77blas.h  lapacke_config.h  lapacke.h  lapacke_mangling.h  lapacke_utils.h  openblas_config.h
```
Thank you for your time!
Sean

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /home/sdrobert/.pyenv/versions/miniconda3-latest/envs/test:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main
blas                      1.0                    openblas
libgcc-ng                 9.1.0                hdf63c60_0
libgfortran-ng            7.3.0                hdf63c60_0
libopenblas               0.3.10               h5a2b251_0
nomkl                     3.0                           0
openblas-devel            0.3.10                        0
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : test
    active env location : /home/sdrobert/.pyenv/versions/miniconda3-latest/envs/test
            shell level : 1
       user config file : /home/sdrobert/.condarc
 populated config files : /home/sdrobert/.condarc
          conda version : 4.9.2
    conda-build version : 3.21.4
         python version : 3.8.8.final.0
       virtual packages : __glibc=2.23=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/sdrobert/.pyenv/versions/miniconda3-latest  (writable)
           channel URLs : https://conda.anaconda.org/pytorch/linux-64
                          https://conda.anaconda.org/pytorch/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/sdrobert/.pyenv/versions/miniconda3-latest/pkgs
                          /home/sdrobert/.conda/pkgs
       envs directories : /home/sdrobert/.pyenv/versions/miniconda3-latest/envs
                          /home/sdrobert/.conda/envs
               platform : linux-64
             user-agent : conda/4.9.2 requests/2.25.1 CPython/3.8.8 Linux/4.4.0-19041-Microsoft ubuntu/16.04.3 glibc/2.23
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False

```
</details>

**Comment by teake** [comment]
I can confirm `openblas-devel` 0.3.10 is broken. `lapack.h` is indeed missing, as the explicit includes in the recipes has not been updated to account for the new header file introduced upstream in https://github.com/xianyi/OpenBLAS/pull/2353) `openblas-devel` 0.3.13 also appears to be broken.

**Comment by katietz** [comment]
I have added this patch to upcoming 0.3.17 version.  As soon as we push it, I will close this issue. Agreed?

**Comment by sdrobert** [comment]
Not sure if you're waiting on me, but agreed :)

**Comment by cbouss** [comment]
Fixed as of 0.3.17
